natten/natten1d.py

```diff
@@ -27,10 +27,14 @@ def __init__(self, dim, kernel_size, num_heads,
         self.kernel_size = kernel_size
         self.dilation = dilation or 1
         self.window_size = self.kernel_size * self.dilation
+        self.with_rpb = with_rpb
 
         self.qkv = nn.Linear(dim, dim * 3, bias=qkv_bias)
-        self.rpb = nn.Parameter(torch.zeros(num_heads, (2 * kernel_size - 1)))
-        trunc_normal_(self.rpb, std=.02, mean=0., a=-2., b=2.)
+        if with_rpb:
+            self.rpb = nn.Parameter(torch.zeros(num_heads, (2 * kernel_size - 1)))
+            trunc_normal_(self.rpb, std=.02, mean=0., a=-2., b=2.)
+        else:
+            self.register_buffer('rpb', torch.zeros(num_heads, (2 * kernel_size - 1)))
```

Comment:
```suggestion
        self.rpb = nn.Parameter(torch.zeros(num_heads, (2 * kernel_size - 1)), requires_grad=with_rpb)
        if with_rpb:
            trunc_normal_(self.rpb, std=.02, mean=0., a=-2., b=2.)
```
Could you format it like this instead?